### PR TITLE
update kvspic and producer c hook

### PR DESCRIPTION
**Pull request by chehefen**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -75,7 +75,7 @@ if(BUILD_DEPENDENCIES)
    build_dependency(gtest)
   endif()
 
-  build_dependency(kvspic)
+  # building kvsCommonLws also builds kvspic
```
Comment (disa6302) [inline CMakeLists.txt:78]: If this is the case, we wouldnt need libkvspic-CMakeLists.txt right?

Comment (chehefen) [reply CMakeLists.txt:78]: removed

**Comment by disa6302** [comment]
Looks like the builds are failing with jsmn errors.

File: CMake/Dependencies/libkvsCommonLws-CMakeLists.txt [hunk 2]
```diff
@@ -6,7 +6,7 @@ include(ExternalProject)
 
 ExternalProject_Add(libkvsCommonLws-download
     GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-producer-c.git
-    GIT_TAG           3c0ffc9dd26483db3f385cc0b38602a0113e327f
+    GIT_TAG           03dd9ea88bc5474878c7023fecfab63f975f6231
```
Comment (disa6302) [inline CMake/Dependencies/libkvsCommonLws-CMakeLists.txt:9]: While you are at it, can you update producer C to this commit? 
270c0b6612cc553d458e316e835471c693f518bd

**Pull request merged**